frontend/model/contracts/chatroom.js

```diff
@@ -176,7 +176,9 @@ sbp('chelonia/defineContract', {
         }
       },
       sideEffect ({ contractID }) {
-        Vue.set(sbp('state/vuex/state')?.chatroom?.chatRoomUnread, contractID, {
+        const chatroomUnread = sbp('state/vuex/state').chatroom?.chatRoomUnread
+        if (!chatroomUnread) return
```

Comment:
I'm rather confused by this...

This `chatroomModule` is supposed to be set/defined. It's part of the initial state for Vuex chatroom module.

And if, for some extraordinary reason (that should definitely generate a `console.warn` at the very least), it is not set, then shouldn't we just proceed with setting it instead of returning immediately?

Reply:
This was causing some issues during testing, although note that the previous version of it was expecting it not to be set sometimes too:

```
sbp('state/vuex/state')?.chatroom?
```

Using `Vue.set(undefined, ...)` is an error, hence the changes.

As for the second part:

> And if, for some extraordinary reason (that should definitely generate a console.warn at the very least),

IMO, this is the wrong place to do this, as it's the application's responsibility to set up `chatroom`. Again IMO, we shouldn't set it here and just treat it as a non-critical error as it not being set doesn't break the contract state.

Reply:
OK, well, a reminder then to output a `console.warn` line